packages/flutter/test/widgets/overscroll_stretch_indicator_test.dart

```diff
@@ -218,6 +223,92 @@ void main() {
     );
   });
 
+  testWidgets('Stretch overscroll works in reverse - horizontal - RTL', (WidgetTester tester) async {
+    final GlobalKey box1Key = GlobalKey();
+    final GlobalKey box2Key = GlobalKey();
+    final GlobalKey box3Key = GlobalKey();
+    final ScrollController controller = ScrollController();
+    await tester.pumpWidget(
+      buildTest(
+        box1Key,
+        box2Key,
+        box3Key,
+        controller,
+        axis: Axis.horizontal,
+        reverse: true,
+        textDirection: TextDirection.rtl,
+      )
+    );
+
+    expect(find.byType(StretchingOverscrollIndicator), findsOneWidget);
+    expect(find.byType(GlowingOverscrollIndicator), findsNothing);
+    final RenderBox box1 = tester.renderObject(find.byKey(box1Key));
+    final RenderBox box2 = tester.renderObject(find.byKey(box2Key));
+    final RenderBox box3 = tester.renderObject(find.byKey(box3Key));
+
+    expect(controller.offset, 0.0);
+    expect(box1.localToGlobal(Offset.zero), Offset.zero);
+    expect(box2.localToGlobal(Offset.zero), const Offset(300.0, 0.0));
+    expect(box3.localToGlobal(Offset.zero), const Offset(600.0, 0.0));
+    await expectLater(
+      find.byType(CustomScrollView),
+      matchesGoldenFile('overscroll_stretch.horizontal.reverse.rtl.start.png'),
+    );
+
+    TestGesture gesture = await tester.startGesture(tester.getCenter(find.byType(CustomScrollView)));
+    // Overscroll the start
+    await gesture.moveBy(const Offset(200.0, 0.0));
+    await tester.pumpAndSettle();
+
+    // expect(box1.localToGlobal(Offset.zero), Offset.zero);
+    // expect(box2.localToGlobal(Offset.zero).dx, greaterThan(305.0));
+    // expect(box3.localToGlobal(Offset.zero).dx, greaterThan(610.0));
```

Comment:
Should these be removed?

Reply:
you are right! but I already tested with the cases uncommented, just uncommented them thank you for noticing.